content/articles/Why-and-when-you-should-use-Kubernetes/index.md

```diff
@@ -0,0 +1,52 @@
+![WHY AND WHEN SHOULD WE USE KUBERNETES?](/Why-and-when-you-should-use-Kubernetes/image-name/container.jpg)
+# WHY AND WHEN SHOULD WE USE KUBERNETES? 
+
+Kubernetes, also known as K8s. Is an open-source system for automating deployment, scaling, and management of containerized applications. Let's dive into the below concepts first. We then look into why and when we should use Kubernetes.
+
+# WHAT IS A CONTAINER
+
+Let us first begin by understanding what containers are. Why so?  Modern applications are being built using containers which are a virtualization environment. Kubernetes is software used to deploy and manage these containers.  Containers are environments that have been virtualized like virtual machines. 
+
+# Similarities Between a Virtual Machine and A container 
+
+One can run many containers on a single virtual machine. This is like a virtual machine. It is possible to set and run many virtual machines on one single machine. This is a physical machine. Docker is an example of a container. It is one of the most popular containers used.  
+
+# Difference Between Virtual Machine and Container.
+
+1) For VM, You will manage the operating system. For Container, there is no management of the operating system. 
+2) For VM, it is an instance of an operating system. For Container, it is lightweight. Meaning containers are built to be started, stopped, and scaled out. This is without human intervention. 
+3) For VM, it is static i.e. as application demand increases, one creates and deploys the virtual machine. For Container, it is more dynamic. They allow one to respond to change in demand. With a container, one can easily and quickly restart it in case of any kind of failure. E.g. Hardware Failure
+
+
+ 
+
+# WHY AND WHEN SHOULD WE USE KUBERNETES? 
```

Comment:
Dont capitalize every word. Please look at how the section headers have been written in the published articles.